include/manifold/cross_section.h

```diff
@@ -164,11 +168,36 @@ class CrossSection {
   static CrossSection Hull(const Polygons polys);
   ///@}
 
+  /** @name Fillet
+   */
+  ///@{
+  std::vector<CrossSection> Fillet(double radius,
```

Comment:
I think this API is plenty - no need for the others as users can already get the same effect using a combination of the existing APIs.